### PR TITLE
Sprint plan ready for review

### PR DESCRIPTION
**Pull request by GijsWeterings**

### CODE REVIEW

**Comment by eanker** [comment]
Looks good to me.

File: Documentation/Sprint 4/Sprint Plan.tex [hunk 1]
```diff
@@ -14,28 +14,133 @@ \section*{Sprint Plan 4}
...
+\item 4 hours
+\end{enumerate}
+\\
+
+As a user, I want to classify songs based on arousal and valence to improve the system.
+
+&
+\begin{enumerate}
+\item Implement frontend function thumbs up and down.
+\item Implement frontend function SAMs.
```
Comment (TimvdLippe) [inline Documentation/Sprint 4/Sprint Plan.tex:56]: We might need to add clarification in paranthesis what SAMs are. (I actually forgot what the abbrevation stands for ;p )

File: Documentation/Sprint 4/Sprint Plan.tex [hunk 2]
```diff
@@ -14,28 +14,133 @@ \section*{Sprint Plan 4}
...
+&
+\begin{enumerate}
+\item Style pages and UI elements
+\item Fix duplication bug in state transformation.
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item Tim van der Lippe
+\item Tim van der Lippe
```
Comment (TimvdLippe) [inline Documentation/Sprint 4/Sprint Plan.tex:124]: I didn't create a Waffle ticket for this one, do we want to create one or was it fixed already @GijsWeterings ?

File: Documentation/Sprint 4/Sprint Plan.tex (hunk 1 above)
Comment (jwgmeligmeyling) [inline Documentation/Sprint 4/Sprint Plan.tex:56]: Software Architecture Models :wink: No, they are Self Assessment Scale.

File: Documentation/Sprint 4/Sprint Plan.tex [hunk 3]
```diff
@@ -14,28 +14,133 @@ \section*{Sprint Plan 4}
...
 \end{tabular}
 \end{table}
+\newpage
 
+\section*{Other activities}
+Sprint planning: 10 hours\\
+Sprint reflection: 10 hours\\
+Daily Scrum Meetings: 8 hours\\
+Reviewing Pull-Requests: 20 hours\\
+Meetings with stakeholders: 2.5 hours\\
```
Comment (TimvdLippe) [inline Documentation/Sprint 4/Sprint Plan.tex:143]: Are we also planning additional meetings with for example Cynthia about the SAMs, or Friso about how to install the application(s)?

**Comment by TimvdLippe** [comment]
Apart from the 3 very minor comments, it looks good to me! Putting all hours on the planning and not only the user stories seems a good update!

File: Documentation/Sprint 4/Sprint Plan.tex (hunk 2 above)
Comment (jwgmeligmeyling) [inline Documentation/Sprint 4/Sprint Plan.tex:124]: The duplication bug is the faulty animation.

Comment (TimvdLippe) [inline Documentation/Sprint 4/Sprint Plan.tex:124]: Yes I know, but I thought Gijs already knew the fix and implemented it (although I couldn't find it). So should I still create a ticket for it or shall we regard it as fixed?

**Comment by TimvdLippe** [comment]
Could someone ( @JWGmeligMeyling probably? ) compile the latex (possibly after the comments) and generate the PDF to put in the SE-Deliverables folder? It does not what to compile on my machines.

**Comment by jwgmeligmeyling** [comment]
How late should we deliver? I can't compile now, but I can compile in the evening.

**Pull request merged**